.github/workflows/test.yml

```diff
@@ -1,25 +1,53 @@
+name: Test
+
 on:
   pull_request:
     branches:
       - master
 
 jobs:
-  build:
+  prepack-upload:
     runs-on: ubuntu-18.04
-    name: Tests
+    name: Prepack and Upload
```

Comment:
Let's just name this `Prepack` the upload / download is an implementation detail, and may change if GitHub rolls out persistent workspaces

Reply:
Will create a separate PR for it.